.github/workflows/test-install-digital-ocean.yaml

```diff
@@ -0,0 +1,83 @@
+name: e2e - DigitalOcean 1-click (install)
+
+on: push
+
+jobs:
+  do-install:
+    runs-on: ubuntu-latest
+    if: github.repository == 'PostHog/charts-clickhouse'
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install doctl
+      uses: digitalocean/action-doctl@v2
+      with:
+        token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+
+    - name: Declare variables that we can share across steps
+      id: vars
+      run: |
+        echo "::set-output name=k8s_cluster_name::helm-e2e-testing-do-1-click-$(git rev-parse --short HEAD)"
+
+    - name: Deploy a new k8s cluster
+      run: |
+        doctl k8s clusters create \
+          ${{ steps.vars.outputs.k8s_cluster_name }} \
+          --region fra1 \
+          --version 1.21.5-do.0 \
+          --tag="provisioned_by:github_action" \
+          --size s-2vcpu-2gb \
+          --count 3 \
+          --wait \
+          --1-clicks \
+          posthog
+
+    # Wait for the DigitalOcean marketplace to complete the installation
+    #
+    # Despite the --wait flag used in the command above
+    # there is no guarantee the PostHog app will be deployed
+    # when the command returns. With this workaround we wait
+    # until the Kubernetes keyspace is created.
+    #
+    # ref: https://github.com/digitalocean/doctl/issues/1063
+    #
+    # Why can't we directly use the 'action-k8s-await-workloads' step below?
+    # Because it's not working for this use case
+    #
+    # ref: https://github.com/jupyterhub/action-k8s-await-workloads/issues/38
+    #
+    - name: Workaround - wait until the 'posthog' pods are ready
+      timeout-minutes: 15
```

Comment:
nit: maybe increase this to 30, sometimes cluster creation/starting Posthog feels quite slow.

Reply:
For what I tested this action usually takes < 3 minutes (see [example](https://github.com/PostHog/charts-clickhouse/runs/4239504054?check_suite_focus=true)). I don't think there's need to increase it to 30 (and 15 is probably too much already).